src/test/kotlin/no/skatteetaten/aurora/boober/utils/asserts.kt

```diff
@@ -89,15 +90,7 @@ fun <T> Assert<Result<T>>.configErrors(messages: List<String>) {
     this.isFailure()
         .isInstanceOf(AuroraConfigException::class)
         .transform { ace ->
-            val errors = ace.errors.map { it.asWarning() }.distinct()
-
-            if (errors.size != messages.size) {
-                this.expected("You do not expect all error messages. Actual error messages are ${errors.size}")
-            }
-            errors.zip(messages).forEach { (actual, expected) ->
-                if (!actual.contains(expected)) {
-                    this.expected(":${show(actual)} to contain:${show(expected)}")
-                }
-            }
+            val errors = ace.errors.map { it.message }.distinct()
+            assertThat(errors).containsExactlyInAnyOrder(*messages.toTypedArray())
```

Comment:
Jeg hadde vært litt forsiktig med å fjerne debugoutput. Det er slitsomt med tester hvor det feiler, men du får ingen feedback på _hva_ som er feilet, og så må du bruke debugging og mye hjernesmurning for å finne ut hvor noe ikke er likt lenger. 

Dette er en metode som brukes _mye_ i testene. Sjekk en håndfull for å se at du ved feil fremdeles får output som gir hjelp til å finne hva som brekker testen.

Reply:
Dette endret jeg fordi jeg ikke fikk nyttig informasjon, bare at antallet var feil. Etter endringen får jeg informasjon om forventet liste, og akkurat hva som kom inn.

Denne metoden brukes bare ett sted, nemlig der jeg endret teksten til å være eksakt det som returneres, og ikke bare en substring.